cmd/machine-config-controller/start.go

```diff
@@ -40,6 +40,7 @@ func init() {
 	rootCmd.AddCommand(startCmd)
 	startCmd.PersistentFlags().StringVar(&startOpts.kubeconfig, "kubeconfig", "", "Kubeconfig file to access a remote cluster (testing only)")
 	startCmd.PersistentFlags().StringVar(&startOpts.resourceLockNamespace, "resourcelock-namespace", metav1.NamespaceSystem, "Path to the template files used for creating MachineConfig objects")
+	startCmd.PersistentFlags().StringVar(&startOpts.promMetricsURL, "metrics-url", "127.0.0.1:8797", "URL for prometheus metrics listener")
```

Comment:
strictly speaking, it's an address, not a URL so `metrics-listen-addres` would be more correct.

Reply:
Updated to reference listen address. This was copied more or less verbatim from the machine-config-daemon, and while I'd like to change it there too, I don't know if anyone actually uses that parameter over there (we don't), so I'm going to lave that one for now until I'm sure it's not going to break anyone.